deployment/ansible/group_vars/all.example

```diff
@@ -10,13 +10,15 @@ pfb_aws_batch_tilemaker_job_queue_name: "name_of_batch_tilemaker_job_queue"
 pfb_aws_batch_tilemaker_job_definition_name: "staging-pfb-tilemaker-run-job"
 
 
-# application settings
-docker_compose_version: 1.8.0
-
 # azavea.docker
 # 1.12.* to match AWS ECS
 docker_version: 1.12.*
+docker_py_version: 1.9.0
+docker_compose_version: 1.8.0
 docker_options: "--storage-opt dm.basesize=20G"
+docker_keyserver: hkp://p80.pool.sks-keyservers.net:80
 
 # azavea.terraform
 terraform_version: 0.9.1
+
+postgresql_database_name: "pfb"
```

Comment:
I don't know that this is being used. It doesn't look like the `azavea.postgresql` role is in the `roles.yml` file.